packages/wallet/src/KeyManagement/util/mapHardwareSigningData.ts

```diff
@@ -86,6 +128,320 @@ const bytesToIp = (bytes?: Uint8Array) => {
   return null;
 };
 
+const prepareTrezorInputs = (
+  inputs: CSL.TransactionInputs,
+  inputAddressResolver: ResolveInputAddress,
+  knownAddresses: GroupedAddress[]
+): CardanoInput[] => {
+  const trezorInputs = [];
+  for (let i = 0; i < inputs.len(); i++) {
+    const input = inputs.get(i);
+    const coreInput = cslToCore.txIn(input);
+    const paymentAddress = inputAddressResolver(coreInput);
+
+    let trezorInput = {
+      prev_hash: Buffer.from(input.transaction_id().to_bytes()).toString('hex'),
+      prev_index: input.index()
+    } as CardanoInput;
+
+    let paymentKeyPath = null;
+    if (paymentAddress) {
+      const knownAddress = knownAddresses.find(({ address }) => address === paymentAddress);
+      if (knownAddress) {
+        paymentKeyPath = [
+          HARDENED + 1852,
```

Comment:
Would be good to have Cardano purpose' and coin_type' constants exported from KeyManagement module, as I think those are also used elsewhere in this package